### DISCUSSION TITLE
React : target container is not a dom element on mobile

### DISCUSSION THREAD

**Discussion by AnthonyDaSilvaFe** [codesandbox/codesandbox-client#4997] [Q&A]
Hello guys, I have this error (target container is not a dom element on mobile) when I try to view my sandbox on mobile.
Any idea?

**Comment by lbogdan** [comment]
Hello @AnthonyDaSilvaFe ,

Can you please provide a link to the sandbox in question, so that we can take a look? Thanks!

**Reply by AnthonyDaSilvaFe** [reply]
https://codesandbox.io/s/htstranslator-n3p04, is this enought ?
Thanks

**Reply by lbogdan** [reply]
Hm, it looks like we can't access it. Is it a private sandbox?

**Reply by AnthonyDaSilvaFe** [reply]
Yes, I'll set it to public

**Reply by AnthonyDaSilvaFe** [reply]
Done

**Reply by lbogdan** [reply]
I can't replicate, it works for me in latest Chrome on iOS 14:

![sandbox_mobile](https://user-images.githubusercontent.com/2083930/95758799-909ec100-0cb1-11eb-8ecb-4ec7446d9545.jpg)

What OS / browser are you using?

**Reply by lbogdan** [reply]
It works in Safari, too.

**Reply by AnthonyDaSilvaFe** [reply]
I'm using chrome on Android and this is what I get.
![Screenshot_20201012-164744](https://user-images.githubusercontent.com/33628633/95760172-f9366f80-0caa-11eb-8345-7f9d989f5826.jpg)

**Comment by lbogdan** [comment]
Tested in latest Chrome on Android, unfortunately I still can't reproduce:

![sandbox_android](https://user-images.githubusercontent.com/2083930/95761516-1839ff00-0cb5-11eb-9308-855bce930fab.png)

**Reply by AnthonyDaSilvaFe** [reply]
Ok, I managed to make it work by opening it in another window. If I'm using the view in the editor, it's not working.

Thanks for your help ^^

**Comment by lbogdan** [comment]
That's weird; as you saw above, for me it worked in both the editor / embed and standalone. Glad that you somehow solved it.

What can I say, browsers tend to be very temperamental at times. 🤷‍♂️

**Reply by AnthonyDaSilvaFe** [reply]
Yeah it's strange and it's pretty random when I tried this morning both embed and the standalone view didn't work.

**Reply by lbogdan** [reply]
Can you maybe try on another mobile device?

**Reply by AnthonyDaSilvaFe** [reply]
Tried on my wife's phone and a coworker's and it's working for them I don't get it ahah.

**Reply by AnthonyDaSilvaFe** [reply]
And now it's working on mine xD
